AppKit/CPView.j

```diff
@@ -1900,10 +1900,12 @@ var CPViewFlags                     = { },
             CPDOMDisplayServerSetStyleSize(_DOMImageParts[0], size.width, size.height);
         }
         else
+        {
             _DOMElement.style.background = colorCSS;
 
             if (patternImage)
```

Comment:
Looks like it should be part of the else block (and thus this was a bug), since lines 1891-1892 test for patternImage also and call CPDOMDisplayServerSetStyleBackgroundSize with very different args.